Sample-01/src/app/pages/external-api/external-api.component.html

```diff
@@ -1,12 +1,65 @@
 <div class="container mt-5">
   <h1>External API</h1>
+
+  <p class="lead">Ping an external API by clicking the button below.</p>
+
+  <p>
+    This will call a local API on port 3001 that would have been started if you
+    run <code>npm run dev</code> (or <code>npm run server:api</code>). An access token is sent as part of the
+    request's `Authorization` header and the API will validate it using the
+    API's audience value.
+  </p>
+
+  <ng-container *ngIf="!audience">
+    <div class="alert alert-warning" role="alert">
+      <p>
+        You can't call the API at the moment because your application does not
```

Comment:
Sounds like an error most API QS could run into. Wouldn't it be better if we put this as part of an "include" file in docs/quickstarts rather than duplicating it across sample apps (and having to update multiple places) ?

Reply:
This is for users downloading the sample app. It happens regularly that users have problems with the configuration and get stuck.

This message should make things a bit clearer for them to understand what is going on.

@stevehobbsdev has done the same work with React after user feedback https://github.com/auth0-samples/auth0-react-samples/pull/222